mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_job_log_entries_function.sql

```diff
@@ -30,15 +30,15 @@ CREATE OR REPLACE FUNCTION get_job_log_entries(jobExecutionId uuid, sortingField
 AS $$
 DECLARE
     v_sortingField text DEFAULT sortingfield;
-    v_entityAttribute text DEFAULT 'marc';
+    v_entityAttribute text[] DEFAULT ARRAY[upper(entityType)];
 BEGIN
 -- Using the source_record_order column in the array type provides support for sorting invoices and marc records.
     IF sortingField = 'source_record_order' THEN
         v_sortingField := 'source_record_order_array';
     END IF;
 
-    IF entityType <> 'ALL' THEN
-        v_entityAttribute := lower(entityType);
+    IF entityType = 'MARC' THEN
+        v_entityAttribute := ARRAY['MARC_BIBLIOGRAPHIC', 'MARC_HOLDINGS', 'MARC_AUTHORITY'];
```

Comment:
Hi @Abdulkhakimov! Can we use rule_type (MARC_BIB instead of MARC_BIBLIOGRAPHIC) from srm instead of hardcode here? Or create new type for this particular case.

Reply:
I would like to use "LIKE" operator to replace hardcode, but it will be affect to performance,. What do you think?

Reply:
I guess, since there is no new upcoming types, performance is more important.